cmd/directories.go

```diff
@@ -44,7 +44,7 @@ func ParseFromCmdArgs(flags *flag.FlagSet, options *DirectoryOptions) error {
 		return err
 	}
 
-	currentDir, err := os.Getwd()
+	currentDir, err := os.Executable()
```

Comment:
Why Getwd doesn't fit ?

Reply:
I made command runnable from any location in filesystem:
```
./build/client/mysterium_client
```

Reply:
so basically current dir is a path to executable ?

Reply:
Yes, exactly